cmd/tuf/app/key-pop-sign.go

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2023 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"flag"
+	"fmt"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/sigstore/sigstore/pkg/signature/options"
+)
+
+func KeyPOPSign() *ffcli.Command {
+	var (
+		flagset   = flag.NewFlagSet("tuf key-pop-sign", flag.ExitOnError)
+		challenge = flagset.String("challenge", "", "the challenge to sign, for a delegate this is the delegate name")
```

Comment:
I think this makes sense? It's the unique delegation rolename, which we want the signer to attest "posession" over.